extension/arquillian-drone-browserstack-extension/README.adoc

```diff
@@ -0,0 +1,138 @@
+= Arquillian Drone BrowserStack extension
+This is an Arquillian extension that enables you to use the BrowserStack as a Drone WebDriver there in your functional tests.
+
+=== How to use it:
+
+1):: First of all, you need to have this extension on your project's classpath. Eg. as a maven dependency:
+[source,xml]
+----
+    <dependency>
+        <groupId>org.jboss.arquillian.extension</groupId>
+        <artifactId>arquillian-drone-browserstack-extension</artifactId>
+        <version>${version.org.jboss.arquillian.drone}</version>
+    </dependency>
+----
+
+2):: To use the BrowserStack driver you need to specify *browserstack* as a *browser* property:
+[source,xml]
+----
+    <extension qualifier="webdriver">
+        <property name="browser">browserstack</property>
+        ...
+    </extension>
+----
+
+3):: You can choose if you want to specify your credentials to access your account or to provide the URL directly with the credentials included.
```

Comment:
I would add also the possibility to specify this username and access.key on the command line via some environment property. In case you run this somewhere, you can not commit your credentials to repository. You just want to have it parametrized during runtime.